compliance/disallowed_images/disallowed_cloud_images.pt

```diff
@@ -0,0 +1,260 @@
+name "Disallowed Cloud Images"
+rs_pt_ver 20180301
+type "policy"
+short_description "Checks for any running instances that are using disallowed cloud images with the option to Terminate them after approval. \n See the [README](https://github.com/flexera/policy_templates/tree/master/compliance/disallowed_images) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+category "Compliance"
+severity "low"
+
+###############################################################################
+# Permissions
+###############################################################################
+
+permission "perm_read_creds" do
+  actions   "rs_cm.show_sensitive","rs_cm.index_sensitive"
+  resources "rs_cm.credentials"
+end
+
+###############################################################################
+# User inputs
+###############################################################################
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses to notify"
+  description "Email addresses of the recipients you wish to notify when new incidents are created."
+end
+
+parameter "param_exclude_tags" do
+  type "list"
+  label "Tags to ignore"
```

Comment:
The label should be Exclude tags.  The description is fine.

Reply:
I have updated to Exclude tags